app/config/variables.php

```diff
@@ -61,6 +61,15 @@
                 'question' => 'Enter your Appwrite hostname',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_DOMAIN_FUNCTIONS',
+                'description' => 'A domain to use for function preview URLs. Setting to "disabled" turns off function preview URLs.',
+                'introduction' => '',
+                'default' => 'disabled',
+                'required' => false,
+                'question' => '',
+                'filter' => ''
+            ],
```

Comment:
@Meldiron we miss basic documentation of the new env vars.

Reply:
Only VCS vars seems to be missing. they also need to be renamed to use `_APP` prefix I believe.

Reply:
Renamed to _APP. Added VCS ones.